Index: docker-compose.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>version: \"3.9\"\r\n\r\nservices:\r\n  db:\r\n    image: postgres:14\r\n    container_name: practicando \r\n    environment:\r\n      - POSTGRES_DB=practica\r\n      - POSTGRES_PASSWORD=pass\r\n      - POSTGRES_USER=user\r\n    ports:\r\n      - \"3456:5432\"\r\n    volumes:\r\n      - local_pgdata:/var/lib/postgresql/data\r\n\r\n  pgadmin:\r\n    image: dpage/pgadmin4\r\n    container_name: practicando-pgadmin\r\n    ports:\r\n      - \"80:80\"\r\n    environment:\r\n      PGADMIN_DEFAULT_EMAIL: roger@xtec.com\r\n      PGADMIN_DEFAULT_PASSWORD: roger123\r\n    volumes:\r\n      - pgadmin-data:/var/lib/pgadmin\r\n\r\nvolumes:\r\n  local_pgdata:\r\n  pgadmin-data:\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/docker-compose.yml b/docker-compose.yml
--- a/docker-compose.yml	(revision 3999edad406cecdb2bf793e46919ccf09e22df0a)
+++ b/docker-compose.yml	(date 1748871258985)
@@ -3,7 +3,7 @@
 services:
   db:
     image: postgres:14
-    container_name: practicando 
+    container_name: practicandofastapi
     environment:
       - POSTGRES_DB=practica
       - POSTGRES_PASSWORD=pass
@@ -15,7 +15,7 @@
 
   pgadmin:
     image: dpage/pgadmin4
-    container_name: practicando-pgadmin
+    container_name: practicandofastapi-pgadmin
     ports:
       - "80:80"
     environment:
