Index: docker-compose.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>version: \"3.9\"\r\n#\r\nservices:\r\n  db:\r\n    image: postgres:14\r\n    container_name: examenuf2recu\r\n    environment:\r\n      - POSTGRES_DB=fastapirecu\r\n      - POSTGRES_PASSWORD=pass\r\n      - POSTGRES_USER=user\r\n    ports:\r\n      - \"3456:5432\"\r\n    volumes:\r\n      - local_pgdata:/var/lib/postgresql/data\r\n\r\n  pgadmin:\r\n    image: dpage/pgadmin4\r\n    container_name: fastapirecu-pgadmin\r\n    ports:\r\n      - \"80:80\"\r\n    environment:\r\n      PGADMIN_DEFAULT_EMAIL: ah39aziz@gmail.com\r\n      PGADMIN_DEFAULT_PASSWORD: ahmed123\r\n    volumes:\r\n      - pgadmin-data:/var/lib/pgadmin\r\n\r\nvolumes:\r\n  local_pgdata:\r\n  pgadmin-data:\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/docker-compose.yml b/docker-compose.yml
--- a/docker-compose.yml	(revision e6fe8080ffe24ec209bc82fc7dac69e015c10cf0)
+++ b/docker-compose.yml	(date 1748871087510)
@@ -19,7 +19,7 @@
     ports:
       - "80:80"
     environment:
-      PGADMIN_DEFAULT_EMAIL: ah39aziz@gmail.com
+      PGADMIN_DEFAULT_EMAIL: ah39@gmail.com
       PGADMIN_DEFAULT_PASSWORD: ahmed123
     volumes:
       - pgadmin-data:/var/lib/pgadmin
